api/assets/resources.json

```diff
@@ -1554,6 +1584,139 @@
         ],
         "description": "This is the main endpoint for accessing your properties. View\nall of the available query parameters and make requests below!\nThe API uses Basic Authentication, which most HTTP libraries\nwill handle for you. To use the test data (which is what this\npages uses), you can use the api key `simplyrets` and secret\n`simplyrets`. Note that these test listings are not live RETS\nlistings but the data, query parameters, and response bodies\nwill all work the same.\n"
```

Comment:
Should this say properties or agents?  Seems to be directed at properties.
Also, do we have test data for agents?

Reply:
This specific line is indeed for the `/properties` endpoint. GitHub just showed it in the diff since it was right above everything I added